### ISSUE TITLE
website/admin: add default theme switching in admin panel

### ISSUE THREAD

**Issue by excii** [R-a-dio/valkyrie#71]
It would be nice to be able to change the default theme from the website for holidays, or for any other reason.

- Permissions for changing or open to everyone with access?

-

**Comment by Wessie** [comment]
site-wide default theme should probably just be under admin permissions, djs can only change their own theme

**Comment by icxes** [comment]
Something to consider is that (iirc) currently we just set every DJ's theme to whatever theme we want to be the "default", which obviously isn't really an option if/when we do DJ themes (in fact it was only fine on current site because we don't have DJ themes). On holidays, holiday themes should still overwrite DJ themes, but probably not user-selected themes, and this should all work without touching the theme_id column in the djs table

So, in other words, there needs to be a way to force some theme "temporarily" without touching the djs table like we are now, and it needs to have priority over whatever the database says that dj's theme should be.

**Comment by Wessie** [comment]
this is now implemented, currently sits under the `admin` permission, but should probably get a self-contained permission instead. the holiday theme is adjustable from the admin home page
